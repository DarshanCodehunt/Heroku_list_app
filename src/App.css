body{
    margin:0px;
    padding:0px;

}

.header-box{
    padding: 20px;
    background: #FF6602;
    font-size: 12px;
    font-weight: 600;
}

.header-box a{
    text-decoration: none;
    color: black;
}

.main-block div:nth-child(even){
 background-color: #E6E6DF;
}

.main-block div:nth-child(odd){
    background-color: #F6F6EF;
}

.story-block{
    display: flex;
    flex-wrap: nowrap;
    flex-direction: row;
    font-family: sans-serif;
    text-align: right;
}

.story-block p {
    padding: 5px;
    margin: 5px;
    font-size: 14px;
    
}

.story-block p span{
    font-size: 11px;
    color: grey;
}

.flex-item1{
    flex:0 0 2%;
}

.flex-item2{
    flex:0 0 5%;
}

.flex-item3{
    text-align: left;
}

.flex-item3 a{
    cursor: pointer;
    text-decoration: none;
    font-size: 11px;
    color:black;
}

.upvote-label{
    font-size: 14px !important; 
    color:black !important;
}

.fa{
    margin-right: 5px !important;
    cursor: pointer;
}

.color-change{
    color:orangered !important;
}

@media(max-width:768px){
  .story-block{
        display: flex;
        flex-wrap: nowrap;
        flex-direction: column-reverse;
        font-family: sans-serif;
        text-align: left;
        line-height: 1.5;
        
    }

    .story-block p{
        font-size: 16px;
    }

    .flex-item1{
        display: none;
    }
}