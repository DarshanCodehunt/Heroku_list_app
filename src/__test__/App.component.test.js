import React from 'react';
import { create } from 'react-test-renderer';
import App from '../App.component';
import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import { Provider } from 'react-redux';



const mockStore = configureMockStore()


describe('<App> component', () => {
    test('Matches the snapshot', () => {
        const store = mockStore([thunk]);
        const AppPage = create(<Provider store={store}>
            <App />
        </Provider>)
        expect(AppPage.toJSON()).toMatchSnapshot()
    })
})