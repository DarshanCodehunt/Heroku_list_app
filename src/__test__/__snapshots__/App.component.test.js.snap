// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> component Matches the snapshot 1`] = `
<div>
  <div
    className="header-box"
  />
  <div
    className="main-block"
  />
</div>
`;
